src/lib_json/json_value.cpp

```diff
@@ -840,6 +840,7 @@ bool Value::isConvertibleTo(ValueType other) const {
            (type() == realValue && InRange(value_.real_, 0, maxUInt)) ||
            type() == booleanValue || type() == nullValue;
   case realValue:
+    return isNumeric() || type() == booleanValue || type() == nullValue;
```

Comment:
We could also use the `[[fallthrough]]` attribute here to document the intent instead of duplicating code, and that would shut PVS Studio up.
https://en.cppreference.com/w/cpp/language/attributes/fallthrough

I think the duplication here is probably an improvement though. It's almost coincidental that convertibility to realValue and convertibility to booleanValue are the same condition.